src/omnisharp/server.ts

```diff
@@ -321,6 +321,7 @@ export class OmniSharpServer {
 
         try {
             let launchResult = await launchOmniSharp(cwd, args, launchInfo, this.platformInfo, options);
+            this.eventStream.post(new ObservableEvents.ShowOmniSharpChannel());
```

Comment:
This would show the OmniSharp log everytime the user launches C# for VS Code? I think that would be very annoying. I would recommend against that behavior.